Resources/migration.yml

```diff
@@ -801,4 +801,10 @@ MachineFrame: CMMachineFrame
 Autolathe: CMAutolathe
 Paper: CMPaper
 Pen: CMPen
-LuxuryPen: CMPenFountain
\ No newline at end of file
+LuxuryPen: CMPenFountain
+
+# 2024-09-13
+#TargetHuman: RMCTargetHuman
+#TargetSyndicate: RMCTargetSyndicate
+#TargetClown: RMCTargetClown
+#TargetStrange: RMCTargetStrange
```

Comment:
Is this intended to be commented out?

Reply:
It causes errors when not commented out and the IDs are correct so I think it may be something on the map side?

Reply:
Do you have the error?

Reply:
This error was explicitly for migrations.yml
```
  Error Message:
   Robust.Shared.Utility.DebugAssertException : RMCTargetHuman is not an entity prototype.
```

Reply:
That's because it is marked as abstract, see here https://github.com/RMC-14/RMC-14/pull/4040#discussion_r1768071544

Reply:
Thanks.